static/js/compose_actions.js

```diff
@@ -55,20 +56,49 @@ exports.set_focus = function (msg_type, opts) {
     }
 };
 
+function compose_not_subscribed_warning() {
+    const stream_name = compose_state.stream_name();
+    const sub = stream_data.get_sub(stream_name);
+    if (sub) {
+        const warning_text = render_compose_not_subscribed({
+            should_display_sub_button: sub.should_display_subscription_button,
+        });
+        $("#compose-send-status")
+            .removeClass(common.status_classes)
+            .addClass("home-error-bar")
+            .attr("stay", "true")
+            .stop(true)
+            .fadeTo(0, 1);
+        $("#compose-error-msg")[0].innerHTML = warning_text;
```

Comment:
For consistency, this could be written as `$("#compose-error-msg").html(warning_text);` using jQuery.

Reply:
Actually, I used the `.html()` method earlier but when running the lint (`tools/lint`)
it was showing this error
```bash
js        | Setting HTML content with jQuery .html() can lead to XSS security bugs.  Consider .text() or using rendered_foo as a variable name if content comes from handlebars and thus is already sanitized. at static/js/compose_actions.js line 72:
js        |         $("#compose-error-msg").html(warning_text);
```
And as the `.text()` method does not accept HTML I have used this trick.

Reply:
As the error message says, you can rename `warning_text` to `rendered_warning_text`, since this comes from a `handlebars` file.

Reply:
Ohh! Missed it. 
Thanks